autogluon_zeroshot/simulation/ensemble_selection_config_scorer.py

```diff
@@ -34,10 +33,17 @@ def __init__(self,
                  max_fold: Optional[float] = None,
                  backend: str = 'native',
                  use_fast_metrics: bool = True,
+                 proxy_fit_metric_map: Optional[Union[dict, str]] = 'default',
```

Comment:
Can we rather do the following:
```suggestion
                 proxy_fit_metric_map: Optional[dict] = None,
```
and then
```
        if proxy_fit_metric_map is None:
            proxy_fit_metric_map = {'roc_auc': 'log_loss'}
```
as it is the standard way of dealing with defaults (otherwise it is very confusing, as passing None is also a default case)

Reply:
Updated to None